src/tests/testdata/floodgate/floodgate_history_300_10F.yaml

```
@@ -0,0 +1,501 @@
+  :white: komadokun_depth9+54aeb169b945faee05176ab69a1e3f16
+  :winner: komadokun_depth9+54aeb169b945faee05176ab69a1e3f16
+  :loser: sample1-1+81dc9bdb52d04dc20036dbd8313ed055
+- :game_id: wdoor+floodgate-300-10F+910+UHJK+20250524130006
+  :black: 910+e205ee2a5de471a70c1fd1b46033a75f
+  :white: UHJK+ed3fd8f59b912285dd10c68ce106f1be
+  :winner: UHJK+ed3fd8f59b912285dd10c68ce106f1be
+  :loser: 910+e205ee2a5de471a70c1fd1b46033a75f
+- :game_id: wdoor+floodgate-300-10F+sinden3_770kai_473stb_10m+Frieren+20250524130010
+  :black: sinden3_770kai_473stb_10m+707d4f98d9d2620cdaab58f19d02a2e4
+  :white: Frieren+171293841d3fdd4af2e12426ce202ac9
+  :winner: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)